scripts/firedrake-install

```diff
@@ -1427,7 +1427,7 @@ def build_and_install_netgen():
     log.info("Installing NGSolve/Netgen.")
     run_pip(["install", "netgen-mesher==6.2.2304.post142.dev0"])
     log.info("Installing ngsPETSc.")
-    url = "git+https://github.com/NGSolve/ngsPETSc.git@main"
+    url = "git+https://github.com/mscroggs/ngsPETSc.git@mscroggs/ufl-legacy2"
```

Comment:
This should be looked at.

Reply:
This is the branch that https://github.com/NGSolve/ngsPETSc/pull/7 is suggesting to merge. Should be reset once that merge is done

Reply:
I can't see any way to get firedrake-install to use a branch of ngsPETSc without this change

Reply:
This is sensible, I will poke @UZerbinati who might be able to give us a status update.

Reply:
https://github.com/NGSolve/ngsPETSc/pull/7 is merged, so resolving this